src/hb-ot-cmap-table.hh

```diff
@@ -56,6 +56,18 @@ struct CmapSubtableFormat0
 	out->add (i);
   }
 
+  void collect_mapping (hb_set_t *unicodes, /* OUT */
+                        hb_map_t *mapping /* OUT */) const
+  {
+    for (unsigned i = 0; i < 256; i++)
+      if (glyphIdArray[i])
+      {
+        hb_codepoint_t glyph = glyphIdArray[i];
+        *unicodes << i;
+        *mapping << hb_pair_t<hb_codepoint_t, hb_codepoint_t> (i, glyph);
```

Comment:
Umm.  Please don't use the `<<` operator when you are dealing with a set or map.  Those should be used only if you are taking a templated type.  Same in other places.  Thanks.